core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -291,6 +291,9 @@ public void applyTableRename(TableToken token, TableToken updatedTableToken) {
         if (token.isWal()) {
             tableSequencerAPI.applyRename(updatedTableToken);
         }
+        if (updatedTableToken.isMatView()) {
```

Comment:
why don't we handle table rename here?

Reply:
Sorry, I don't understand, table renames are handled here as far as I can see. There is only a special case when the renamed "object" is a mat view, then we need to update the token in mat view graph with the token that has updated table logical name and same dir name.